.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug Report
+about: Something is not working
+---
+
+<!--
+   Thank you for submitting an issue. Please fill in the template below
+   information about the bug you encountered.
+-->
+
+#### Summary
+<!-- Please explain the bug in a few short sentences -->
+
+#### What Should Happen Instead?
+<!-- Please explain what the expected behavior is -->
+
+#### Reproduction Steps
+<!-- Are you able to consistently reproduce the issue? Please add a list of steps that lead to the bug. -->
+
+1. ...
+2. ...
+
+#### System information
+<!-- Please provide an overview of your setup (e.g. number of nodes) and the output of: 
+
+`snap version`
+`uname -a`
+`snap list k8s`
+`snap services k8s`
+`journalctl --lines 200 -f -u snap.k8s.k8sd`
```

Comment:
this does not include important logs from the kubernetes services or containerd. how about something like `snap logs k8s -n 10000`?